.github/workflows/ci.yml

```
@@ -0,0 +1,53 @@
+name: ci-with-docker
+
+on:
+  push: # 아래 브랜치에 Push되면 작동
+    branches: [ "develop" ]
+  pull_request: # 아래 브랜치에 PullRequest 되면 작동
+    branches: [ "develop" ]
+
+permissions:
+  contents: read
+
+# 작업들
+jobs:
+  # 작업 명(본인이 지으면 됨)
+  build:
+    environment: dev
+    runs-on: ubuntu-latest # 대여해주는 서버
+    steps:
+      - uses: actions/checkout@v3 # 라이브러리 이용
+      - name: Set up JDK 17 # JDK 17버전 사용
+        uses: actions/setup-java@v3 # 라이브러리 이용
+        with:
+          java-version: '17' # 지바 버전에 따라 다르게 설정 필요
+          distribution: 'temurin'
+
+      - name: Build with Gradle # 빌드 작업
+        uses: gradle/gradle-build-action@bd5760595778326ba7f1441bcf7e88b49de61a25 # v2.6.0
```

```suggestion
        uses: gradle/gradle-build-action@v2
```

이렇게 적어주면 major 버전 하위에서 최신버전을 선택합니다~

---

하위에서 최신버전 선택하는거 좋네요!! 수정했습니다!!

수정 : [36f975e](https://github.com/prgrms-be-devcourse/BE-04-TICKETPARIS/pull/6/commits/36f975e9894b194dd215c45f4dc2cb94a9b445f4)